src/Take.Blip.Builder.UnitTests/LazyInputTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Lime.Protocol;
+using NSubstitute;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Lime.Protocol.Serialization;
+using Shouldly;
+using Take.Blip.Builder.Models;
+using Takenet.Iris.Messaging.Resources.ArtificialIntelligence;
+using Xunit;
+using Input = Take.Blip.Builder.Models.Input;
+using Newtonsoft.Json;
+using Lime.Protocol.Serialization.Newtonsoft;
+
+#pragma warning disable 4014
+
+namespace Take.Blip.Builder.UnitTests
+{
+    public class LazyInputTests : FlowManagerTestsBase, IDisposable
+
+    {
+        [Fact]
+        public async Task ArtificialIntelligenceExtensionAnalyzeAsyncShouldBeExecuted()
```

Comment:
To keep method name consistent with other tests rename to something like: 

MessageWithAnalyzableMetadataShouldExecuteArtificialIntelligenceAnalyse

Reply:
Method name changed